web/docs/tql2/operators/save_tcp.md

```diff
@@ -0,0 +1,59 @@
+# save_tcp
+
+Saves bytes to a TCP or TLS connection.
+
+```tql
+save_tcp endpoint:string, [tls=bool, certfile=string, keyfile=string]
+```
+
+## Description
+
+Saves bytes to the given endpoint via TCP or TLS.
+
+### `endpoint: string`
+
+The endpoint to which the server will connect. Must be of the form
+`[tcp://]<hostname>:<port>`.
+
+### `tls = bool (optional)`
+
+Whether to use TLS.
+
+Defaults to `false`.
+
+### `skip_peer_verification = bool (optional)`
```

Comment:
What do we want to name this option? In `to_splunk` we use `no_tls_verify`. Ultimately, I think we should use the same name for this option for all operators that do TLS things.

Reply:
@mavam suggested we use the same name as the `*_http` operators. I was going to ask about the splunk one.